### ISSUE TITLE
Add first commit date to initial repository data

### ISSUE THREAD

**Issue by Relequestual** [json-schema-org/ecosystem#5]
Depends on: https://github.com/json-schema-org/ecosystem/issues/4

Add to repository data with first commit date (which is a passable proxy for when JSON Schema related project was created)

Location: https://github.com/json-schema-org/ecosystem/tree/main/projects/initial-data

**Comment by litvinovajann** [comment]
@Relequestual 
Hello! May I start working on this?

**Comment by Relequestual** [comment]
Closing in favour of #11
